docs/setup.rst

```diff
@@ -193,6 +193,11 @@ Docker Method
      container and thus the one of the consumption directory. Furthermore, you
      can change the id of the default user as well using ``USERMAP_UID``.
 
+  ``USE_SSL``
```

Comment:
This should be `PAPERLESS_USE_SSL`, shouldn't it?

Reply:
Of course.